packages/terra-arrange/src/Arrange.module.scss

```diff
@@ -37,3 +37,16 @@
     align-self: flex-start;
   }
 }
+
+.fit-block {
+  .icon-wrapper {
+    display: inline-flex;
+    font-size: 8px;
```

Comment:
Are these style values provided by UX..??

Reply:
No, these are not provided by UX team. I have added them to match the existing look of UI. If required i will check with UX team.

Reply:
yes we need UX review for the above style changes !!

Reply:
Got +1 from UX